README.md

```diff
@@ -34,76 +34,57 @@ sed -i "s/02:FF:FF:FF:FF:FF/$mac_oui:FF:FF:FF/" kubemacpool.yaml
 kubectl apply -f ./kubemacpool.yaml
 ```
 
-### Opting-in to kubemacpool service
+### kubemacpool service
 
-Kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=allocate` - to opt in pods mac allocation in your namespace
-- `mutatevirtualmachines.kubemacpool.io=allocate` - to opt in vms mac allocation in your namespace
+Kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms using two webhooks: mutatepods and mutatevirtualmachines.
+While in [testing manifests](config/test/kubemacpool.yaml) both webhooks are set to be in opt-out mode, [release manifests](config/release/kubemacpool.yaml) set mutatepods to opt-in and mutatevirtualmachines to opt-out.
```

Comment:
Let's not get straight to "how". Say that user can specify whether MAC address will be allocated to all VMs and Pods by default or not. This can be done by patching of the manifests.

Reply:
Done